meta-refkit/recipes-image/images/refkit-installer-image.bb

```diff
@@ -23,10 +23,14 @@ REFKIT_INSTALLER_UEFI_COMBO () {
         rootfs=$4
         output_mounted=
         output_mountpoint=
+        output_luks=
+        LUKS_NAME=rootfs
+        LUKS_PASSWORD=refkit
```

Comment:
The password feature is always enabled, but never used unless explicitly enabled by setting FIXED_PASSWORD=yes when calling image-installer.

Are you suggesting to not put the code into the image-installer script by default and once enabled, print a warning? That seems counter-intuitive - after all, the feature would have been enabled intentionally in such a setup.